non-critical.json

```diff
@@ -2,7 +2,6 @@
   "extends": [
     ":dependencyDashboard",
     ":prHourlyLimit2",
-    ":prNotPending",
```

Comment:
This is the only change I added. The rest in this file was migrated automatically by Renovate